be/src/common/config.h

```diff
@@ -1318,6 +1318,8 @@ CONF_mInt64(datacache_mem_adjust_period, "20");
 CONF_mInt64(datacache_mem_adjust_interval_seconds, "10");
 
 CONF_Int32(datacache_inline_item_count_limit, "130172");
+CONF_Int32(datacache_ttl_check_interval_ms, "0");
+CONF_Int32(datacache_ttl_remove_batch, "100");
```

Comment:
These configurations are the underlying configuration of cache library, and users usually do not need to modify it in starrocks, and most users do not know how to modify it, so there is no need to expose it.

Reply:
The cache production speed is different.  If users can't custom the two configurations. This may cause the cache production rate to be faster than the deletion rate

Reply:
> The cache production speed is different. If users can't custom the two configurations. This may cause the cache production rate to be faster than the deletion rate

I think how to coordinate these two parameters is quite confusing for most users, so we can set a reasonable default value for `datacache_ttl_remove_batch` and only keep `datacache_ttl_check_interval_ms` for adjustment. This can satisfy most user cases.

Reply:
> > The cache production speed is different. If users can't custom the two configurations. This may cause the cache production rate to be faster than the deletion rate
> 
> I think how to coordinate these two parameters is quite confusing for most users, so we can set a reasonable default value for `datacache_ttl_remove_batch` and only keep `datacache_ttl_check_interval_ms` for adjustment. This can satisfy most user cases.

Good advice, fixed